src/lando/main/models/profile.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import annotations
+
+from django.contrib.auth.models import User
+from django.db import models
+
+from lando.main.models.base import BaseModel
+
+
+class Profile(BaseModel):
```

Comment:
This is pretty typical/conventional naming, so that you can access the profile via `user.profile`. `user.lando_user` is a little more awkward, and having "user" in there is redundant. The `User` model itself is the Lando user.